src/core/directives/google-map-marker.ts

```diff
@@ -71,6 +71,11 @@ export class SebmGoogleMapMarker implements OnDestroy,
   iconUrl: string;
 
   /**
+   * Whether to automatically open the info window when the marker is clicked.
+   */
+  openInfoWindow: boolean = true;
```

Comment:
@SebastianM Can I maybe try to convince you to not change the name? I agree with changing the comment to make it clearer, but `openChildInfoWindow` is a bit verbose for the marker property don't you think?

```js
<sebm-google-map-marker [openChildInfoWindow]=true>
```

The way I see it, when we're dealing with a marker, it's implicit that the info window is it's child.  In fact, naming it that leads me to believe that there might be a non-child InfoWindow... meh just being pedantic with the verbiage, your call, I'll address it tomorrow morning before work thanks man!